### PR TITLE
feat: inject tailwind and global styles

### PR DESCRIPTION
**Pull request by HeartSquared**
## Important: Request PR reviews on Slack
Please reach out to the design system team on Slack in `#prod_design_system` for PR reviews. GitHub notifications (e.g. from tagging a person) are not actively monitored.

### CODE REVIEW

File: packages/components/bin/addTailwindStylesImport.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  const filePath = path.resolve(dirOrFile)
+  if (fs.statSync(filePath).isFile()) {
+    const sourceFile = dirOrFile
+      .replace(/dist\/(esm|cjs)/, "src")
+      .replace(/(c|m)js$/, "tsx")
+    if (!fs.existsSync(sourceFile)) {
+      return
+    }
+
+    const stylesPath = path.relative(path.dirname(filePath), DIST_STYLES_PATH)
```
Comment (HeartSquared) [inline packages/components/bin/addTailwindStylesImport.ts:26]: Note: Use scoped path `@kaizen/components` to stop agadoo failing

File: packages/components/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
   "module": "dist/esm/index.mjs",
   "types": "dist/types/index.d.ts",
   "sideEffects": [
+    "tailwind.css",
```
Comment (gyfchong) [inline packages/components/package.json:26]: Actually how come this is necessary? Consumers don't need to import this, and it seems like the components can access it fine

Comment (HeartSquared) [reply packages/components/package.json:26]: Tested in unified-home and it was tree-shaken lol

Comment (gyfchong) [reply packages/components/package.json:26]: Ohh, this key needs a better name lol

**Pull request closed without merging**